web/app.go

```diff
@@ -166,24 +166,23 @@ func NewAppWithDeps(config *Config, deps Dependencies) (*App, error) {
 	webEngine.StaticFS("/static", http.FS(assetsFS))
 	webEngine.GET("/", HomeHandler)
 	webEngine.GET("/about", NewAboutHandler(deps.subscriptionsService))
-	webEngine.GET("/hosts", NewHostListHandler(deps.consul, deps.tagsService))
-	webEngine.GET("/hosts-next", NewHostListNextHandler(deps.hostsNextService))
+	webEngine.GET("/hosts", NewHostListNextHandler(deps.hostsService))
```

Comment:
Is intentional to keep the `Next` word here?

Reply:
thanks it is a leftover im going to address it in a separate PR